build.gradle

```diff
@@ -697,6 +698,34 @@ project(":samza-kv-rocksdb_$scalaSuffix") {
   }
 }
 
+project(":samza-kv-remote_$scalaSuffix") {
```

Comment:
Should this be samza-kv-couchbase?

Reply:
We are thinking to use samza-kv-remote as the package for all remote table providers.

Reply:
Why? Is there something shared across all providers? If so, does it makes sense to separate them into two packages - samza-kv-remote and samza-kv-couchbase? If we add another remote table provider in the future, it doesn't make sense to require users to pull in couchbase dependencies as well.

Reply:
What you said made sense. What I thought was that keeping the remote table providers together makes it easy for users to find them. But I don't have strong opinion on this since I am not familiar with how Samza organizes modules. @weisong44 How do you think about it?

Reply:
Yeah makes sense, let's make the change.